saleor/discount/migrations/tasks/saleor3_19.py

```diff
@@ -0,0 +1,27 @@
+from ....celeryconf import app
+from ... import DiscountType
+from ...models import CheckoutLineDiscount, OrderLineDiscount
+
+BATCH_SIZE = 250
+
+
+@app.task
+def update_discount_type_order_line_task():
+    order_lines = OrderLineDiscount.objects.filter(type=DiscountType.PROMOTION)[
```

Comment:
We can just get only ids, no need to get from DB whole `OrderLineDiscount` object. This way you may be able to increase BATCH_SIZE.